example/lib/main.dart

```diff
@@ -63,42 +64,49 @@ class _MyHomePageState extends State<MyHomePage> {
     child: new Icon(Icons.person, color:Colors.amber,),
   );
 
-  List<Event> _markedDateMap = [
-    new Event(
-      date: new DateTime(2018, 12, 10),
-      title: 'Event 1',
-      icon: _eventIcon,
-    ),
-    new Event(
-      date: new DateTime(2018, 12, 10),
-      title: 'Event 2',
-      icon: _eventIcon,
-    ),
-    new Event(
-      date: new DateTime(2018, 12, 10),
-      title: 'Event 3',
-      icon: _eventIcon,
-    ),
-    new Event(
-      date: new DateTime(2018, 12, 10),
-      title: 'Event 4',
-      icon: _eventIcon,
-    ),
-    new Event(
+  Events _markedDateMap = new Events(
+    events : {
+      new DateTime(2018, 12, 10) : [
+        new Event(
+          date: new DateTime(2018, 12, 10),
+          title: 'Event 1',
+          icon: _eventIcon,
+        ),
+        new Event(
+          date: new DateTime(2018, 12, 10),
+          title: 'Event 2',
+          icon: _eventIcon,
+        ),
+        new Event(
+          date: new DateTime(2018, 12, 10),
+          title: 'Event 3',
+          icon: _eventIcon,
+        ),
+        new Event(
+          date: new DateTime(2018, 12, 10),
+          title: 'Event 4',
+          icon: _eventIcon,
+        ),
+      ],
+      new DateTime(2018, 12, 25) : [
+        new Event(
         date: new DateTime(2018, 12, 25),
         title: 'Event 5',
         icon: _eventIcon,
-    ),
-  ];
+        ),
+      ],
+    },
+  );
 
   CalendarCarousel _calendarCarousel, _calendarCarouselNoHeader;
 
   @override
   Widget build(BuildContext context) {
     /// Example with custom icon
     _calendarCarousel = CalendarCarousel(
-      onDayPressed: (DateTime date) {
+      onDayPressed: (DateTime date, List<Event> events) {
```

Comment:
Could be use `Events` class instead `List<Event>` here?

Reply:
No, if use events class, you need to call getEvents with DateTime parameter. So it would be easier to pass the value only

Reply:
I've resolved the first one because this was related. Ok. if you are sure about this let's leave this. Will resolve this too. Thank you!